src/sentry/consumers/__init__.py

```diff
@@ -0,0 +1,20 @@
+from typing import Mapping, TypedDict
+
+from django.conf import settings
+
+
+class ConsumerDefinition(TypedDict):
+    topic: str
+    strategy_factory: str
+
+
+# consumer group -> consumer definition
+KAFKA_CONSUMERS: Mapping[str, ConsumerDefinition] = {
+    "ingest-profiles": {
+        "topic": settings.KAFKA_PROFILES,
+        "strategy_factory": "sentry.profiles.consumers.process.factory.ProcessProfileStrategyFactory",
+    }
+}
```

Comment:
This structure seems good to me though let's not refer to the key here as a "consumer group" which means something very different in Kafka? How about we call this like a "Sentry consumer name" or something else less overloaded?

Reply:
I renamed consumer group to consumer name